src/DDTrace/Transport/Http.php

```diff
@@ -68,10 +68,13 @@ private function configure($config)
         ], $config);
     }
 
-    public function send(array $traces)
+    /**
+     * {@inheritdoc}
+     */
+    public function send(Tracer $tracer)
     {
-        $tracesPayload = $this->encoder->encodeTraces($traces);
-        self::logDebug('About to send to the agent {count} traces', ['count' => count($traces)]);
+        $tracesPayload = $this->encoder->encodeTraces($tracer);
+        self::logDebug('About to send trace(s) to the agent');
```

Comment:
One thing I don't get is how we replicate the old behavior. Before if we had "unfinished" traces not sent to the agent. How can we replicate this?

Reply:
I'm not sure which old functionality you're referring to with the unfinished spans count, but if you think we should be able to count the traces publicly, we could have `DDTrace\Contracts\Tracer` [implement `Countable`](https://www.php.net/countable) if you think that'd be a good idea. :)

Reply:
FTR, the main reason I didn't go this route is to not introduce another BC, but if you think it's important enough, we should totally add it now since the major version is still `0`. :)

Reply:
Nevermind. This is done in `Tracer:getTracesAsArray()` 👍